src/Codegen/Types.hack

```diff
@@ -112,6 +100,22 @@ function get_input_class(string $hack_type): ?string {
     return null;
 }
 
+/**
+ * Get the GraphQL type to output for a hack type.
+ */
+function get_output_type(string $hack_type): ?string {
```

Comment:
Now that we don't have separate input/output classes, this switch statement (+enum type handling) could be shared between output_type and input_type

Reply:
Good point!